### ISSUE TITLE
can't understand the mean of code in SVM kernel?

### ISSUE THREAD

**Issue by trangtv57** [nfmcclure/tensorflow_cookbook#53]
Can you explain more detail in code SVM kernel, in loss function and predict, i try to read more about SVM to understand but can't, or you can send me some blog with math( in your code) it will better for me. Thank you so much.

**Comment by nfmcclure** [comment]
Hi @trangtv57 ,

Thanks for your interest.  I would recommend reading any/all jupyter notebooks in the SVM categories.

Also, read my response to another similar SVM issue here (It has links to more reading about SVMs):

https://github.com/nfmcclure/tensorflow_cookbook/issues/7

Good luck!
